rotkehlchen/chain/ethereum/tokens.py

```diff
@@ -33,10 +33,100 @@
+    Asset('eip155:1/erc20:0x3a3A65aAb0dd2A17E3F1947bA16138cd37d08c04'),
+    Asset('eip155:1/erc20:0xD37EE7e4f452C6638c96536e68090De8cBcdb583'),
+    Asset('eip155:1/erc20:0x9D91BE44C06d373a8a226E1f3b146956083803eB'),
+    Asset('eip155:1/erc20:0x39C6b3e42d6A679d7D776778Fe880BC9487C2EDA'),
+    Asset('eip155:1/erc20:0x7D2D3688Df45Ce7C552E19c27e007673da9204B8'),
+    Asset('eip155:1/erc20:0xA64BD6C70Cb9051F6A9ba1F163Fdc07E0DfB5F84'),
+    Asset('eip155:1/erc20:0xa06bC25B5805d5F8d82847D191Cb4Af5A3e873E0'),
+    Asset('eip155:1/erc20:0x6FCE4A401B6B80ACe52baAefE4421Bd188e76F6f'),
+    Asset('eip155:1/erc20:0xa685a61171bb30d4072B338c80Cb7b2c865c873E'),
+    Asset('eip155:1/erc20:0x7deB5e830be29F91E298ba5FF1356BB7f8146998'),
+    Asset('eip155:1/erc20:0xc713e5E149D5D0715DcD1c156a020976e7E56B88'),
+    Asset('eip155:1/erc20:0x69948cC03f478B95283F7dbf1CE764d0fc7EC54C'),
+    Asset('eip155:1/erc20:0xCC12AbE4ff81c9378D670De1b57F8e0Dd228D77a'),
+    Asset('eip155:1/erc20:0x71010A9D003445aC60C4e6A7017c1E89A477B438'),
+    Asset('eip155:1/erc20:0x328C4c80BC7aCa0834Db37e6600A6c49E12Da4DE'),
+    Asset('eip155:1/erc20:0x35f6B052C598d933D69A4EEC4D04c73A191fE6c2'),
+    Asset('eip155:1/erc20:0x625aE63000f46200499120B906716420bd059240'),
+    Asset('eip155:1/erc20:0x6C5024Cd4F8A59110119C56f8933403A539555EB'),
+    Asset('eip155:1/erc20:0x4DA9b813057D04BAef4e5800E36083717b4a0341'),
+    Asset('eip155:1/erc20:0x101cc05f4A51C0319f570d5E146a8C625198e636'),
+    Asset('eip155:1/erc20:0x9bA00D6856a4eDF4665BcA2C2309936572473B7E'),
+    Asset('eip155:1/erc20:0xBcca60bB61934080951369a648Fb03DF4F96263C'),
+    Asset('eip155:1/erc20:0x71fc860F7D3A592A4a98740e39dB31d25db65ae8'),
+    Asset('eip155:1/erc20:0x3Ed3B47Dd13EC9a98b44e6204A523E766B225811'),
+    Asset('eip155:1/erc20:0xFC4B8ED459e00e5400be803A9BB3954234FD50e3'),
+    Asset('eip155:1/erc20:0x9ff58f4fFB29fA2266Ab25e75e2A8b3503311656'),
+    Asset('eip155:1/erc20:0x030bA81f1c18d280636F32af80b9AAd02Cf0854e'),
+    Asset('eip155:1/erc20:0x12e51E77DAAA58aA0E9247db7510Ea4B46F9bEAd'),
+    Asset('eip155:1/erc20:0x5165d24277cD063F5ac44Efd447B27025e888f37'),
+    Asset('eip155:1/erc20:0x6Fb0855c404E09c47C3fBCA25f08d4E41f9F062f'),
+    Asset('eip155:1/erc20:0xDf7FF54aAcAcbFf42dfe29DD6144A69b629f8C9e'),
+    Asset('eip155:1/erc20:0xB124541127A0A657f056D9Dd06188c4F1b0e5aab'),
+    Asset('eip155:1/erc20:0xB9D7CB55f463405CDfBe4E90a6D2Df01C2B92BF1'),
+    Asset('eip155:1/erc20:0x272F97b7a56a387aE942350bBC7Df5700f8a4576'),
+    Asset('eip155:1/erc20:0xc9BC48c72154ef3e5425641a3c747242112a46AF'),
+    Asset('eip155:1/erc20:0xF256CC7847E919FAc9B808cC216cAc87CCF2f47a'),
+    Asset('eip155:1/erc20:0x1982b2F5814301d4e9a8b0201555376e62F82428'),
+    # Maker tokens
+    Asset('eip155:1/erc20:0x6B175474E89094C44Da98b954EedeAC495271d0F'),
+    Asset('eip155:1/erc20:0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2'),
+    Asset('eip155:1/erc20:0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2'),
```

Comment:
Is this the list of collaterals + dai? we have `create_collateral_type_mapping` in the makerdao module so maybe we can get the assets from there